infra/modules/providers/azure/cosmosdb/variables.tf

```diff
@@ -48,4 +48,10 @@ variable "databases" {
     throughput = number
   }))
   default = []
+}
+
+# VNET Restriction (optional)
```

Comment:
If optional, do we need a default variable?

Reply:
added `"10.2.0.0/16"` as default